CONTRIBUTING.md

```diff
@@ -688,9 +688,10 @@ Unless:
             # Create a new environment with new cursor database
             new_env = api.Environment(new_cr, self.env.uid, self.env.context)
             # with_env replace original env for this method
+            # A good comment here of because is required this transaction isolated.
```

Comment:
Full sentence should be "A good comment here of why this isolated transaction is required."